src/components/KeyboardShortcut.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import classNames from 'classnames'
+
+type Modifier = 'ctrl' | 'alt' | 'shift'
+type TranslateModifiers = (modifier: Modifier) => string
+
+function defaultTranslateModifiers(modifier: Modifier) {
+    return modifier.charAt(0).toUpperCase() + modifier.slice(1).toLowerCase()
+}
+
+function hasModifiers(str: string) {
+    return /\b(mod|cmd|ctrl|alt|shift)\b/.test(str)
```

Comment:
I figured that since in macOS we always express `cmd` as `⌘` it does not need translation. Indeed, when on macOS the translate modifiers flow is not used. And since `cmd` is always expressed as `ctrl` in non-macOS environments, that translation is enough.